karma.conf.ts

```diff
@@ -29,17 +29,16 @@
 const webpackConfig = require("./webpack.config.js");
 const tsconfig = require("./tsconfig.json");
 
-import { Config, ConfigOptions } from "karma";
-
 const testRecursivePath = "test/**/*.ts"
     , srcOriginalRecursivePath = "src/**/*.ts"
     , srcRecursivePath = "lib/**/*.js"
     , srcCssRecursivePath = "lib/**/*.css"
     , coverageFolder = "coverage";
 
 process.env.CHROME_BIN = require("puppeteer").executablePath();
-module.exports = (config: Config) => {
-    config.set(<ConfigOptions>{
+module.exports = (config) => {
+    config.set({
+        mode: "development",
```

Comment:
remove it

Reply:
Was used for debugging tests. Now removed.